### ISSUE TITLE
Enhancing automatic span names based on Classes names

### ISSUE THREAD

**Issue by LikeTheSalad** [elastic/apm-agent-android#69]
Now:
co.elastic.apm.opbeans.modules.customers.CustomersFragment - View appearing
Proposing:
CustomersFragment (co.elastic.apm.opbeans.modules.customers) - View appearing

**Comment by felixbarny** [comment]
Maybe even just `CustomersFragment - View appearing`? That would be similar to how we name Transactions in the Java agent (`RestControlerClassName#methodName`).

**Comment by LikeTheSalad** [comment]
My concern with it is that it's not impossible to have 2 classes with the same simple name but in a different package name, so if that ever happens, we might get those bucketed as the same transaction type?

**Comment by felixbarny** [comment]
That's correct but the same concern also applies to the way the Java agent groups transactions. So far, we didn't get complaints.

**Comment by LikeTheSalad** [comment]
I see, I guess it should be fine to use the simple name only then, to be honest from an aesthetic point of view I prefer only using the simple name as well, so I think it's good news that nobody has complained so far! Will do the same here then
